piwebapi_samples/JQuery/azure-pipelines.yml

```diff
@@ -48,28 +48,9 @@ jobs:
           fileLibrary: $(CredFileLibrary)
           filePath: $(projPath)
 
-      - task: NodeTool@0
-        inputs:
-          versionSpec: '10.x'
-        displayName: 'Specify Node version'
-
-      - script: |
-          echo NPM CI
-          call npm ci
-          echo Install Karma CLI
-          call npm install -g karma-cli
-          echo Run tests
-          call karma start
-          echo Complete
-        workingDirectory: $(Build.SourcesDirectory)$(testPath)
-        displayName: 'Run tests'
-
-      - task: PublishTestResults@2
-        inputs:
-          testResultsFormat: 'JUnit'
-          testResultsFiles: '**\TEST*.xml'
-          searchFolder: $(Build.SourcesDirectory)$(testPath)
-        displayName: 'Publish test results'
+      - template: '/piwebapi_samples/JQuery/run-tests.yml'
+        parameters:
+          projectPath: $(Build.SourcesDirectory)$(projPath)
```

Comment:
Looks like this should be `$(testPath)` at the end, not `$(projPath)`. 
From the original, `workingDirectory: $(Build.SourcesDirectory)$(testPath)`

Reply:
Fixed path.